db/schema/amountDetail.ts

```diff
@@ -0,0 +1,23 @@
+import mongoose, { Schema } from 'mongoose';
+
+const amountDetailSchema: Schema = new mongoose.Schema({
+  id: String,
+  transactionId: {
+    type: 'string',
+    nullable: true,
+  },
+  transactionPersonId: {
+    type: 'string',
+    nullable: true,
+  },
+  amount: Number,
+  subAmount: Number,
+  tax: Number,
+  currency: {
```

Comment:
Same with currency: we should have this in a separate schema or modulate somehow since this is being used in item schema as well.

Reply:
Sounds good to me, especially with how we're going to be adding all the currencies eventually, it might be cleaner to separate it out, will do for currency and status